Makefile.toml

```diff
@@ -0,0 +1,213 @@
+env_files = [
+    { path = ".env/mainnet.env", profile = "mainnet" },
+    { path = ".env/testnet.env", profile = "testnet" },
+    { path = ".env/local.env", profile = "local" },
+    { path = ".env/custom.env", profile = "custom" },
+    { path = ".env/local.env", profile = "development" },
+]
+
+[config]
+default_to_workspace = false
+skip_core_tasks = true
+
+[env]
+CARGO = "cargo"
+ENGINE_CARGO_TARGET = "wasm32-unknown-unknown"
+SWEEP_DAYS = 30
+
+[tasks.sweep]
+category = "Cleanup"
+install_crate = "cargo-sweep"
+command = "${CARGO}"
+args = [
+    "sweep",
+    "--time",
+    "${SWEEP_DAYS}",
+]
+
+[tasks.clean-cargo]
+category = "Cleanup"
+command = "${CARGO}"
+args = ["clean"]
+
+[tasks.clean-contracts]
+category = "Cleanup"
+script = '''
+rm -Rf etc/eth-contracts/.pnp
+rm -Rf etc/eth-contracts/artifacts
+rm -Rf etc/eth-contracts/cache
+rm -Rf etc/benchmark-contract/target
+'''
+
+[tasks.clean-bin]
+category = "Cleanup"
+script = '''
+rm -Rf bin
+'''
+
+[tasks.clean-flow]
+category = "Cleanup"
+dependencies = [
+    "clean-cargo",
+    "clean-contracts",
+    "clean-bin",
+]
+
+[tasks.clean]
+run_task = "clean-flow"
+
+[tasks.fmt]
+category = "Formatting"
+command = "${CARGO}"
+args = [
+    "fmt",
+    "--all",
+]
+
+[tasks.clippy]
+category = "Check"
+command = "${CARGO}"
+args = [
+    "clippy",
+    "--no-default-features",
+    "--features",
+    "contract",
+    "--",
+    "-D",
+    "warnings",
+]
+
+[tasks.check-fmt]
+category = "Check"
+command = "${CARGO}"
+args = [
+    "fmt",
+    "--",
+    "--check",
+]
+
+[tasks.check-contracts]
+category = "Check"
+script = '''
+cd etc/eth-contracts
+yarn
+yarn lint
+'''
+
+[tasks.check]
+category = "Check"
+dependencies = [
+    "check-contracts",
+    "check-fmt",
+    "clippy",
+]
+
+[tasks.post-engine-build-env]
+category = "Tools"
+script = '''
+echo "Environment:"
+echo "    IS_RELEASE:           ${IS_RELEASE}"
+echo "    CARGO_FEATURES:       ${ACTUAL_FEATURES}"
+echo "    TARGET_WASM:          ${ACTUAL_WASM}"
+echo "    TARGET_DIR:           ${TARGET_DIR}"
+echo "    Extra build args:     ${@}"
```

Comment:
Unclear how it works in that case

Reply:
It is the last flow that is ran so we can be visually certain that the engine is compiling correctly at the end of the build.